sceptre/cli/template.py

```diff
@@ -66,7 +87,11 @@ def generate_command(ctx, path):
     )
 
     plan = SceptrePlan(context)
-    responses = plan.generate()
+
+    execution_context = null_context() if no_placeholders else use_resolver_placeholders_on_error()
```

Comment:
it looks like validate and diff contains a more verbose version for this logic.  just curious, any reason why it's different here?

Reply:
A previous version had parameters passed to use_resolver_placeholders_on_error but then they were removed. I can make them uniform now, though.